openmapflow/notebooks/compare_ceo_labels.ipynb

```diff
@@ -0,0 +1,2563 @@
+{
```

Comment:
<p>minor nit: I think some of the cells in this notebook could be squashed into one cell and like this and the above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/openmapflow/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>